src/libcore/sync/atomic.rs

```diff
@@ -160,14 +160,14 @@ pub fn spin_loop_hint() {
 /// This type has the same in-memory representation as a [`bool`].
 ///
 /// [`bool`]: ../../../std/primitive.bool.html
-#[cfg(target_has_atomic = "8")]
+#[cfg(any(target_has_atomic_load_store = "8", target_has_atomic = "8"))]
```

Comment:
This repetition seems rather ungreat. Why not just `#[cfg(target_has_atomic_load_store = "8")]`? -- If `target_has_atomic = "X"` holds then surely so should the `_load_store` version also.

Reply:
See my other comment from 10 minutes ago: it's because `target_has_atomic_load_store` isn't available in stage0.

Reply:
In that case `cfg{_attr}(bootstrap)` should be used so that we can simplify this once 1.40 becomes the bootstrap compiler.

Reply:
Done.